SQL_module/SubjectForm/db_manipulation.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+function executeQuery($sql)
+    {
+        $host = "localhost";
+        $dbname = "universitySystem";
+        $username = "root";
+        $pass = "";
+        $table_name = "electives";
+
+        try {
+            $connection = new PDO("mysql:host=$host;dbname=$dbname;charset=utf8", $username, $pass, array(PDO::MYSQL_ATTR_INIT_COMMAND => "SET NAMES utf8"));
+            $connection->query($sql);
+
+            printContentOfTable($connection, $table_name);
+        }
+        catch(PDOException $error) {
+            echo $error->getMessage();
+        }
+    }
+
+    function insert($title, $description, $lecturer){
+        executeQuery("INSERT INTO electives(title,description,lecturer) VALUES ('$title', '$description', '$lecturer')");
```

Comment:
Now that I think about this, I think the current code is vulnerable to sql injection as it does not use prepared statements.